.github/ISSUE_TEMPLATE/feature-request.yaml

```diff
@@ -0,0 +1,67 @@
+name: Feature Request
+description: File a feature request or task
+title: "[FTR]: "
+labels: ["triage"]
+projects: ["Team Engineering Project"]
```

Comment:
The `projects` field does not seem to be allowed in the YAML schema for issue templates.

```suggestion
```